frontend/server/src/main/java/com/amazonaws/ml/mms/ServerInitializer.java

```diff
@@ -27,15 +28,17 @@
  */
 public class ServerInitializer extends ChannelInitializer<SocketChannel> {
 
+    private final boolean controlAPI;
     private SslContext sslCtx;
 
     /**
      * Creates a new {@code HttpRequestHandler} instance.
      *
      * @param sslCtx null if SSL is not enabled
      */
-    public ServerInitializer(SslContext sslCtx) {
+    public ServerInitializer(SslContext sslCtx, boolean controlAPI) {
```

Comment:
Maybe, controlAPI --> separateControlPlanePort ?
Also, update the function comments with the additional param

Reply:
I renamed it to controlPlaneServer since it initializes a controlPlaneServer vs an APIServer.  I don't want to include the word separate as it implies that there is an option to have a combined API.